api/src/test/java/com/epam/pipeline/test/web/AbstractControllerTest.java

```diff
@@ -59,12 +59,12 @@
             "multipart/form-data; boundary=--------------------------boundary";
     protected static final String MULTIPART_CONTENT =
             "----------------------------boundary\r\n" +
-                    "Content-Disposition: form-data; name=\"file\"; filename=\"file.txt\"\r\n" +
-                    "Content-Type:  application/octet-stream\r\n" +
-                    "\r\n" +
-                    "file.txt" +
-                    "\r\n" +
-                    "----------------------------boundary";
+            "Content-Disposition: form-data; name=\"file\"; filename=\"file.txt\"\r\n" +
+            "Content-Type:  application/octet-stream\r\n" +
+            "\r\n" +
+            "file.txt" +
```

Comment:
I'm wondering if we can extract `MULTIPART_CONTENT_FILE_CONTENT`. `MULTIPART_CONTENT_FILE_NAME` and `MULTIPART_CONTENT_FILE_CONTENT` fields. Additionally let's replace the content of the file from `file.txt` to some more distinct string like `content of file.txt`. Once changed please check that no tests depending on `MULTIPART_CONTENT` are failing.